### PR TITLE
Auth: Reduce lookup cookie error to warning and invalidate cookie

### PR DESCRIPTION
**Pull request by Jguer**
**What this PR does / why we need it**:

- Reduce lookup cookie error to warning. 
- Delete invalid cookie to avoid continuous refresh

### CODE REVIEW

File: pkg/services/contexthandler/contexthandler.go [hunk 1]
```diff
@@ -420,6 +423,18 @@ func (h *ContextHandler) initContextWithToken(reqContext *models.ReqContext, org
 	return true
 }
 
+func (h *ContextHandler) deleteInvalidCookieEndOfRequestFunc(reqContext *models.ReqContext) web.BeforeFunc {
+	return func(w web.ResponseWriter) {
+		// if response has already been written, skip.
+		if w.Written() {
+			return
```
Comment (sakjur) [inline pkg/services/contexthandler/contexthandler.go:430]: Maybe debug log this path too?

**Review by kalleep — APPROVED** [review]
LGTM

**Comment by jmatosgrafana** [comment]
LGTM from a security point of view

**Review by jmatosgrafana — APPROVED** [review]
LGTM from a security point of view

**Pull request merged**